modules/aleph/src/Aleph/Handler/AlephPatronHandler.php

```diff
@@ -55,7 +55,8 @@ public function __construct(AlephClient $client, AlephPatron $patron = NULL) {
    */
   public function authenticate($bor_id, $verification) {
     $response = $this->client->authenticate($bor_id, $verification);
-    $result = new AuthenticationResult($this->client, $bor_id, $verification);
+    $result = new AuthenticationResult($this->client, $bor_id, $verification,
+      aleph_get_allowed_login_branches(), $this->getActiveBranches($bor_id));
```

Comment:
I think this is an unfortunate change. Until now everything in the `Aleph` namespace has been framework agnostic. I consider that a great feature from a code architecture perspective. By calling `aleph_get_allowed_login_branches()` within our code we tie it to Drupal.

These values should either be injected or the remaining structure should be updated so calling `aleph_get_allowed_login_branches()` from within the `Aleph` namespace is no longer necessary.

Reply:
Wooops, that was a mistake! :)